src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/ClassFanOutComplexityCheck.java

```diff
@@ -49,6 +49,7 @@ public ClassFanOutComplexityCheck() {
     public int[] getRequiredTokens() {
         return new int[] {
             TokenTypes.PACKAGE_DEF,
+            TokenTypes.IMPORT,
```

Comment:
It has a common base with `ClassDataAbstractionCouplingCheck` and I decided, that adding `exludedPackages` to the `ClassFanOutComplexityCheck` will be a good idea. Should I implement it for the `ClassDataAbstractionCouplingCheck` only?